docs/reference/commandline/stack_ps.md

```diff
@@ -49,9 +71,196 @@ Multiple filter flags are combined as an `OR` filter. For example,
+STATE            ERROR               PORTS
+w48spazhbmxc        voting_redis.1      redis:alpine        node2               Running             Running
+17 minutes ago
+t72q3z038jeh        voting_redis.2      redis:alpine        node3               Running             Running
+17 minutes ago
+```
+
+#### node
+
+The `node` filter matches on a node name or a node ID.
+
+```bash
+$ docker stack ps -f "node=node1" voting
+ID                  NAME                  IMAGE                                          NODE
+ DESIRED STATE       CURRENT STATE            ERROR               PORTS
+q7yik0ks1in6        voting_result.1       dockersamples/examplevotingapp_result:before   node1
+ Running             Running 18 minutes ago
+tz6j82jnwrx7        voting_db.1           postgres:9.4                                   node1
+ Running             Running 18 minutes ago
+6jj1m02freg1        voting_visualizer.1   dockersamples/visualizer:stable                node1
+ Running             Running 18 minutes ago
+```
+
+#### desired-state
+
+The `desired-state` filter can take the values `running`, `shutdown`, and `accepted`.
+
+```bash
+$ docker stack ps -f "desired-state=running" voting
+ID                  NAME                  IMAGE                                          NODE
+ DESIRED STATE       CURRENT STATE            ERROR               PORTS
+xim5bcqtgk1b        voting_worker.1       dockersamples/examplevotingapp_worker:latest   node2
+ Running             Running 21 minutes ago
+q7yik0ks1in6        voting_result.1       dockersamples/examplevotingapp_result:before   node1
+ Running             Running 21 minutes ago
+rx5yo0866nfx        voting_vote.1         dockersamples/examplevotingapp_vote:before     node3
+ Running             Running 21 minutes ago
+tz6j82jnwrx7        voting_db.1           postgres:9.4                                   node1
+ Running             Running 21 minutes ago
+w48spazhbmxc        voting_redis.1        redis:alpine                                   node2
+ Running             Running 21 minutes ago
+6jj1m02freg1        voting_visualizer.1   dockersamples/visualizer:stable                node1
+ Running             Running 21 minutes ago
+kqgdmededccb        voting_vote.2         dockersamples/examplevotingapp_vote:before     node2
+ Running             Running 21 minutes ago
+t72q3z038jeh        voting_redis.2        redis:alpine                                   node3
+ Running             Running 21 minutes ago
+```
+
+### Formatting
+
+The formatting options (`--format`) pretty-prints tasks output using a Go template.
+
+Valid placeholders for the Go template are listed below:
+
+Placeholder     | Description
+----------------|------------------------------------------------------------------------------------------
+`.ID`           | Task ID
+`.Name`         | Task name
+`.Image`        | Task image
+`.Node`         | Node ID
+`.DesiredState` | Desired state of the task (`running`, `shutdown`, and `accepted`)
```

Comment:
I think this should be "`running`, `shutdown`, or `accepted`" ("or" instead of "and")